superbench/benchmarks/micro_benchmarks/cublas_function.py

```diff
@@ -190,7 +190,7 @@ def add_parser_arguments(self):
         self._parser.add_argument(
             '--num_warmup',
             type=int,
-            default=8,
+            default=8 * 1000,
```

Comment:
I don't understand this change.  The default warm-up iterations should be small or 0.

Reply:
this is to match our initial design for training scenario and to keep it consistent with our previous validation setting.